packages/js/src/dashboard/components/problems.js

```diff
@@ -1,12 +1,50 @@
-import { Paper, Title } from "@yoast/ui-library";
+import { __, _n } from "@wordpress/i18n";
+import { ExclamationCircleIcon } from "@heroicons/react/outline";
+import { Paper } from "@yoast/ui-library";
+import { AlertsList } from "./alerts-list";
+import { AlertTitle } from "./alert-title";
+import { Collapsible } from "./collapsible";
+import { AlertsContext } from "../routes/alert-center";
 
 /**
  * @returns {JSX.Element} The problems component.
  */
 export const Problems = () => {
+	const problemsList = [
+		{
+			message: __( "Huge SEO issue: You're blocking access to robots. If you want search engines to show this site in their results, you must go to your Reading Settings and uncheck the box for Search Engine Visibility. I don't want this site to show in the search results.", "wordpress-seo" ),
+		},
+		{
+			message: __( "You still have the default WordPress tagline, even an empty one is probably better. You can fix this in the customizer.", "wordpress-seo" ),
```

Comment:
let's not make these placeholders translatable since they are going to be removed soon.
Are there other cases?

Reply:
yes, in notifications.js, I fixed it